.circleci/config.yml

```diff
@@ -1,4 +1,13 @@
 version: 2
+workflows:
+  matrix-tests:
+    jobs:
+      - node/test:
```

Comment:
You need to use the circleci node orb to use the `node/test` job

https://circleci.com/developer/orbs/orb/circleci/node